packages/web3-errors/src/errors/generic_errors.ts

```diff
@@ -83,6 +83,12 @@ export class OperationAbortError extends BaseWeb3Error {
 
 export class AbiError extends BaseWeb3Error {
 	public code = ERR_ABI_ENCODING;
+	public readonly props: Record<string, unknown> & { name?: string };
+
+	public constructor(message: string, props?: Record<string, unknown> & { name?: string }) {
+		super(message);
```

Comment:
The base class `BaseWeb3Error` has `innerError` and it accepts it as an optional second parameter in the constructor.

```suggestion
	public constructor(message: string, props?: Record<string, unknown> & { name?: string }, innerError?: Error | Error[]) {
		super(message, innerError);
```

Reply:
Yes